### PR TITLE
Add Netlify Logo

### PR DESCRIPTION
**Pull request by mmcbride2929**
# TL/DR
This PR updates the footer component by adding the netlify logo.

# Overview of Change

`Footer.jsx`
- Add `<a>` & `` tags with netlify link & logo
- Add className to `<a>` tag

`index.css`
- Style `<a>` & `` tags 

netlify-logo

### CODE REVIEW

**Review by readysetagile — CHANGES_REQUESTED** [review]
the footer test is failing.  please fix

```Snapshot Summary
 › 2 snapshots failed from 2 test suites. Inspect your code changes or press `u` to update them.

Test Suites: 2 failed, 6 passed, 8 total
Tests:       2 failed, 12 passed, 14 total
Snapshots:   2 failed, 6 passed, 8 total
Time:        4.47 s
Ran all test suites.
```

File: src/index.css [hunk 1]
```diff
@@ -228,4 +231,14 @@ footer p {
 
 footer p {
   margin: 0;
-}
\ No newline at end of file
+}
+
+footer .image-wrapper {
```
Comment (jduffey) [inline src/index.css:236]: Screenshot 2023-08-02 at 2 54 05 PM

Nit: fix formatting, remove extra whitespace.

Comment (jduffey) [reply src/index.css:236]: Screenshot 2023-08-02 at 2 56 17 PM

fwiw I only first noticed this by doing a local diff of your branch against the `upstream/dev` branch (after making sure to pull in order to be up-to-date). You can see the red in the console, indicating there's extra whitespace. (Not all terminals will be configured to show this necessarily; it's probably a default setting that comes with macOS Xcode tools or some other terminal settings I have applied.)

**Comment by jduffey** [comment]
Overall, looks good. I'll wait till the failing test is resolved before approving.

**Review by jduffey — APPROVED** [review]
The minor formatting issue and failing test have been addressed. Everything looks good on `51453b4`. Approved. :shipit:

**Review by readysetagile — APPROVED** [review]
+1 passed

**Pull request merged**